Package.swift

```diff
@@ -0,0 +1,36 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "PayoneerCheckout",
+    platforms: [
+        .iOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "PayoneerCheckout",
+            targets: ["PayoneerCheckout"])
+    ],
+    dependencies: [
+
+    ],
```

Comment:
Is it needed?

Reply:
I usually like to leave it in because it's part of the generated syntax and we'll most likely use it soon, but I can remove it no problem.

Reply:
Just doesn't look nice :), but that is a matter of taste :). I will resolve that comment because it is more about decoration.

```suggestion
dependencies: [],
```

Reply:
I removed it already, no worries.